packages/tasit-act/src/TasitAct.test.js

```diff
@@ -0,0 +1,137 @@
+import { Contract } from "./TasitAct";
+import Account from "tasit-account";
+import { expect, assert } from "chai";
+import { waitForEvent, mineBlocks } from "./helpers/utils.js";
+
+// Note: SimpleStorage.json is originally genarated by `tasit-contracts` and was pasted here manually
+// See https://github.com/tasitlabs/TasitSDK/issues/45
+import { abi as contractABI } from "./helpers/SimpleStorage.json";
+
+// Note: Under the current `tasit-contracts` setup SimpleStorage aways will deployed with this address
+// See https://github.com/tasitlabs/TasitSDK/pull/59#discussion_r242258739
+const contractAddress = "0x6C4A015797DDDd87866451914eCe1e8b19261931";
```

Comment:
Sorry, I don't' get it.

Reply:
Could you explain that to me?

Reply:
Oh, if it's deterministic that it always gets this address, have logic in here that gets the address programmatically by importing the necessary files from `tasit-contracts` to determine what the address would deterministically be.

That way if we ever changed the setup in `tasit-contracts` so that a new deterministic address were picked, the test wouldn't break here

Reply:
What made me think of this were the comments above this line:
```
// Note: Under the current `tasit-contracts` setup SimpleStorage aways will deployed with this address
// See https://github.com/tasitlabs/TasitSDK/pull/59#discussion_r242258739
```

Reply:
Right, thanks! I've extracted to: https://github.com/tasitlabs/TasitSDK/issues/138

Reply:
Great! Resolving